src/makeCalculator.js

```diff
@@ -37,7 +37,26 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (result, number) => result + number,
+    subtract: (result, number) => result - number,
+    multiply: (result, number) => result * number,
+    divide: (result, number) => result / number,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
```

Comment:
```suggestion
    add(result, number) {
      return result + number
    },
    
    subtract(result, number) {
      return result - number;
    }
    
    multiply(result, number) {
      return result * number;
    }
    
    divide(result, number) {
      return result / number;
    }

    operate(callback, number) {
      this.result = callback(this.result, number);

      return this;
    },
```

Reply:
Arrow functions are great, but let's use one approach for methods creating, some of your methods created by arrow functions, some by function declaration